backend/src/crd/controller/scheduledworkflow/util/scheduled_workflow.go

```diff
@@ -341,7 +341,7 @@ func (s *ScheduledWorkflow) updateLastTriggeredTime(epoch int64) {
 }
 
 func (s *ScheduledWorkflow) updateNextTriggeredTime(epoch int64) {
-	if epoch != math.MaxInt64 {
+	if epoch != math.MaxInt64 && epoch != 1<<63-62135596801 {
```

Comment:
This looks hacky, can we improve? I don't have an idea right now.

Reply:
Updated to check if smaller than maxEpoch, but still kind of hacky.